package.json

```diff
@@ -35,7 +35,6 @@
     "functional-red-black-tree": "^1.0.1",
     "i18n-js": "^3.3.0",
     "node-libs-react-native": "^1.2.1",
-    "qrcode": "^1.4.1",
```

Comment:
Hey @ulisesmac : I think you may have to also check in the `yarn.lock` for this corresponding change.